sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -1262,8 +1262,8 @@ class DatasetSuite extends QueryTest with SharedSparkSession {
 
   test("SPARK-16995: flat mapping on Dataset containing a column created with lit/expr") {
     val df = Seq("1").toDF("a")
-
-    import df.sparkSession.implicits._
```

Comment:
Hm, maybe it's no big deal but seems it's going to break such import style?

Reply:
@HyukjinKwon 
As per comments, now not required to update the DataSetSuite.scala file, so I reverted it.
Thanks for reviewing.